laurentlaurent/ShutterFeed-a-soen-341-repo
# [#49] send pictures to firebase

## Summary

This PR adds a 'New post' button. Once clicked, it opens up a dialog that prompts the user for an image and caption. Once the submit button is pressed, the image is sent to the Firebase storage, and the post information is sent to the Firestore database.

This PR used Material for styling.

## Related Issue



* Resolves #49 Send Pictures to Firebase

## Testing 

1. Checkout branch or go to https://shutterfeed-soen341.firebaseapp.com/
2. Follow the steps to submit an image.
3. Check on Firebase to see if the information properly appears.
